cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldStructuredMetadata.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// API includes
+var jobLogger = require('dw/system').Logger.getLogger('Cloudinary', 'CREATE');
+var Status = require('dw/system/Status');
+
+module.exports.Start = function (args) {
+    var cloudinaryConstants = require('*/cartridge/scripts/util/cloudinaryConstants');
+    var cloudinaryMetadataSvc = require('*/cartridge/scripts/service/cldMetadata');
+    var jobStepHelpers = require('*/cartridge/scripts/helpers/jobStepHelpers');
+
+    if (!cloudinaryConstants.CLD_ENABLED) {
+        return new Status(Status.ERROR, 'ERROR', 'The Cloudinary cartridge is disabled');
+    }
+
+    if (jobStepHelpers.isStepDisabled(args)) {
+        return new Status(Status.OK, 'OK', 'Step disabled, skip it...');
+    }
+
+    const schema = [
+        { "label": "sfcc-product-name", "external_id": "sfcc-product-name", "type": "string" },
+        { "label": "sfcc-view-type", "external_id": "sfcc-view-type", "type": "string" },
+        { "label": "sfcc-product-identifier", "external_id": "sfcc-product-identifier", "type": "string" },
+        { "label": "sfcc-is-main", "external_id": "sfcc-is-main", "type": "enum", "datasource": { "values": [{ "external_id": "True", "value": "True" }, { "external_id": "False", "value": "False" }] } },
```

Comment:
Per the QA feedback on https://rafterone.atlassian.net/browse/CLD-252?focusedCommentId=594186, please add:
`{ "label": "sfcc-product-brand", "external_id": "sfcc-product-brand", "type": "string" },`